pkg/pipeline/transform/kubernetes/kubernetes.go

```diff
@@ -164,6 +166,14 @@ func (k *KubeData) NewNodeInformer(informerFactory informers.SharedInformerFacto
 					ips = append(ips, ip.String())
 				}
 			}
+			// Add IP that is used in OVN for some traffic on mp0 interface
+			ip, err := findOvnMp0IP(node.Annotations)
+			if err != nil {
+				// Log the error as Info, do not block other ips indexing
+				log.Infof("failed to index OVN mp0 IP: %v", err)
```

Comment:
@jotak I am just thinking that this might be totally not relevant if we are not running with OVN CNI, so I suggest moving that log line to debug.

Reply:
this shouldn't be a problem, as if we're not running ovn, we wouldn't have the annotation `k8s.ovn.org/node-subnets` => no error returned in that case

Reply:
ok 👍